data/revised.stc

```diff
@@ -180,27 +180,33 @@ Modify 82257 # Improved spectral class from McCook & Sion
 }
 
 Modify 86032 # Rasalhague: rapidly rotating star, rotational parameters from
-             # Monnier et al. (2017), ApJ 725 (1), 1192
-             # "Rotationally Modulated g-modes in the Rapidly Rotating δ Scuti Star 
-             # Rasalhague (α Ophiuchi)"
-             # https://arxiv.org/abs/1012.0787
+			 # Monnier et al. (2017), ApJ 725 (1), 1192
+			 # "Rotationally Modulated g-modes in the Rapidly Rotating δ Scuti Star 
+			 # Rasalhague (α Ophiuchi)"
+			 # https://arxiv.org/abs/1012.0787
 {
-    Radius 1988300
-    SemiAxes [ 1 0.84 1 ] # equatorial radius 19.7% greater than polar
-
-    UniformRotation
-    {
-        Period         14.55 # from rotation rate
-        Inclination    65.12 # inclination (i) = 87.5 deg
-        AscendingNode 239.23 # orientation (alpha) = -53.5 deg
-    }
+	Radius 1988300
+	SemiAxes [ 1 0.84 1 ] # equatorial radius 19.7% greater than polar
+
+	UniformRotation
+	{
+		Period		 14.55 # from rotation rate
+		Inclination	65.12 # inclination (i) = 87.5 deg
+		AscendingNode 239.23 # orientation (alpha) = -53.5 deg
+	}
 }
 
 Modify 88267 # 95 Her: Spectral class from SIMBAD; replaces "G5"
 {
 	SpectralType "A5III"
 }
 
+Modify 89931 # Del Sgr: Distance from Hipparcos parallax, which is more accurate than the Gaia parallax
+{
+	Distance 347.726 # 9.38 +/- 0.18 mas
```

Comment:
Using a parallax of 9.38 mas (and the "lazy" inversion method) I got 347.715 ly. I know this is a small difference but it's probably best to clear this up to minimize confusion.

Reply:
I used a parsecs-to-light-years conversion factor of 3.26167, which IIRC is Celestia's internal value.

Reply:
Okay, that makes sense. But I wonder why would Celestia use 3.26167? The correct value (when rounded) would be 3.26156. Very strange.